py-utils/src/utils/msg_bus/message_bus/adaptee/adaptee.py

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Lets move this file to message_bus itself. Also Please rename to file and class to "MessageBroker" which is the base class for All the Message Brokers such as Kafka to be written

Reply:
moved